File: web/src/model/victron_controller/dashboard/from_0_1_0_sensor-meta.ts
// @ts-nocheck
import {SensorMeta as v0_1_0_SensorMeta} from './v0_1_0/SensorMeta'
import {SensorMeta as dashboard_SensorMeta} from './SensorMeta'

export function convert__sensor_meta__from__0_1_0(from: v0_1_0_SensorMeta): dashboard_SensorMeta {
    return new dashboard_SensorMeta (
        from.origin,
        from.identifier,
        from.cadence_ms,
        from.staleness_ms
    )
}

File: crates/dashboard-model/src/victron_controller/dashboard/v0_2_0/from_0_1_0_sensor_meta.rs


pub fn convert__sensor_meta__from__0_1_0(from: &crate::victron_controller::dashboard::v0_1_0::sensor_meta::SensorMeta) -> crate::victron_controller::dashboard::v0_2_0::sensor_meta::SensorMeta {
    crate::victron_controller::dashboard::v0_2_0::sensor_meta::SensorMeta {
        origin: from.origin.clone(),
        identifier: from.identifier.clone(),
        cadence_ms: from.cadence_ms.clone(),
        staleness_ms: from.staleness_ms.clone(),
    }
}